pkgs/top-level/python-packages.nix

```diff
@@ -8263,6 +8263,9 @@ self: super: with self; {
     tesseract = pkgs.tesseract5;
   };
 
+  octodns-hetzner = callPackage ../development/python-modules/octodns-hetzner { };
+  octodns-powerdns = callPackage ../development/python-modules/octodns-powerdns { };
```

Comment:
```suggestion
  octodns-hetzner = callPackage ../development/python-modules/octodns-hetzner { };

  octodns-powerdns = callPackage ../development/python-modules/octodns-powerdns { };
```

Reply:
I can change this if this is a blocker for you but would prefer to leave it this way because those packages are quite related and there is multiple instances of this in package sets.